Source/WebCore/loader/ResourceMonitor.cpp

```diff
@@ -61,9 +61,10 @@ void ResourceMonitor::setEligibility(Eligibility eligibility)
         return;
 
     m_eligibility = eligibility;
-    RESOURCEMONITOR_RELEASE_LOG("The frame is %" PUBLIC_LOG_STRING ".", (eligibility == Eligibility::Eligible ? "eligible" : "not eligible"));
 
     if (isEligible()) {
+        RESOURCEMONITOR_RELEASE_LOG("Frame (%" SENSITIVE_LOG_STRING ") was set as eligible.", m_frameURL.utf8().data());
```

Comment:
For logs that are emitted from the WebContent process, you may also consider using `RELEASE_LOG_FORWARDABLE`, which is more efficient.